frontend/packages/operator-lifecycle-manager/src/components/index.tsx

```diff
@@ -114,6 +114,11 @@ export const providedAPIForModel = (csv: ClusterServiceVersionKind, model: K8sKi
 
 export const parseALMExamples = (csv: ClusterServiceVersionKind) => {
   try {
+    if (new URLSearchParams(window.location.search).has('useInitializationResource')) {
```

Comment:
This should be passed as a parameter from caller instead of read directly from the URL here. (This helper shouldn't know about the query parameter, which is specific to a certain page.)